src/components/common/links/ExternalLink.tsx

```diff
@@ -33,7 +33,7 @@ export function ExternalLink(
       rel="noopener noreferrer"
     >
       <span className={styles.content}>{props.children}</span>
-      <Icon icon="open_in_new" className={styles.icon} />
+      <Icon aria-hidden="true" icon="open_in_new" className={styles.icon} />
```

Comment:
```suggestion
      <Icon alt="" icon="open_in_new" className={styles.icon} />
```

go/GAR-web#overview-of-criterion-11 Best practices 3: You can hide decorative or redundant images from assistive technology by setting the alternative text to an empty string (alt="")

I'm not 100% confident that this will work on a font icon though. Please test and inspect the accessibility tree -- I'm curious about the findings

Reply:
If it generates HTML as a <span> or similar then `alt` won't work. Feel free to close this comment in that case

Reply:
Testing this after a long break, the generated HTML is below. Adding alt doesn't prevent the reader from reading the icon, likely because there's some inner text:


`<i aria-hidden="true" class="rmwc-icon rmwc-icon--ligature material-icons _icon_1cowi_23">open_in_new</i>`